### ISSUE TITLE
Set Position for Selve SE-Pro-Com doesn't work

### ISSUE THREAD

**Issue by TwofoldRisk** [Kannix2005/homeassistant-selve#28]
**Describe the bug**
The set to position feature does work for the Selve SE-Pro but not for the SE-Pro-Com.

**To Reproduce**

go to settings - automation and scenes

Set an automatic behovier, 
if temperatur of entity forcaste home is above xx degress 

set device SE pro-com 
action set shuter position to 20 percent

it also doesn't work in the device settings itself

go to settings -> devices and services -> selve NG -> devices -> SE pro-com -> control elemet -> on the windows slide the set to slider doens't get any behavior only moving on clicking the arrows or stop button.

**Expected behavior**
All of the described issue works well with the selve SE pro, don't know why the company installed one se pro and two se pro-com.

**Screenshots**

**Desktop (please complete the following information):**
 - HomeAssistant Version: [2024.8.3]
 - OS [Hassio 13.1]
 - Used shutters [1x SE pro , 2x SE pro-com]

thanks alot for your great work here, and hope to find a solution for this :)

**Comment by Kannix2005** [comment]
Is there any error thrown? Or does it simply not take the position argument in the move call?

**Comment by TwofoldRisk** [comment]
Hi,

It does simply not take the position argument in the move call

Best Regards
Kevin

Von meinem iPhone gesendet

Am 05.09.2024 um 13:02 schrieb Stefan Altheimer ***@***.***>:

﻿

Is there any error thrown? Or does it simply not take the position argument in the move call?

**Comment by Kannix2005** [comment]
Hrm.. both are commeo actors. So this shouldn´t be a problem. Are you sure that the pro coms support a positional move? And did you setup the endstop positions?

**Comment by TwofoldRisk** [comment]
Yeah the Up stop and down stop positions were programmed via a selve cable remote from the window construction company. As far as I understand are the se-com-pro with integrated radio receiving module and on the se-com should be installed an extra radio receiver because the se-com is an cabled actor.

Best
Kevin
Von meinem iPhone gesendet

Am 05.09.2024 um 17:19 schrieb Stefan Altheimer ***@***.***>:

﻿

Hrm.. both are commeo actors. So this shouldn´t be a problem. Are you sure that the pro coms support a positional move? And did you setup the endstop positions?

**Comment by Kannix2005** [comment]
Okay, does setting the position work when using the Windows App?

**Comment by TwofoldRisk** [comment]
Hi there,

If you mean the selve usb rf gatewaytool for windows, there isn‘t a set to option neither for the se-pro nor the se-pro-com.

Here is what i see and can set or control.


Von meinem iPhone gesendet

Am 06.09.2024 um 12:03 schrieb Stefan Altheimer ***@***.***>:

﻿

Okay, does setting the position work when using the Windows App?

**Comment by Kannix2005** [comment]
There´s no picture, if you meant to attach one. But if the Win Tool does not show the option to set a position the shutters dont support it or they are not set up correctly.

**Comment by TwofoldRisk** [comment]
> There´s no picture, if you meant to attach one. But if the Win Tool does not show the option to set a position the shutters dont support it or they are not set up correctly.

But there is no option no matter if se-pro or se-pro-com and on the se-pro it works correctly
![IMG_3183](https://github.com/user-attachments/assets/c76b28eb-19e3-4b1c-a59f-7e5d8dc33dbb)

**Comment by Android551** [comment]
Is it possible to move the shutters to certain positions? For example, I would like to move the shutters to 60% for shading. The commeo actuators send their position back, so it should be possible. It certainly works in the ioBroker selve adapter. But I would like to switch to homeassistant and use the full functionality of the Selve shutters

**Comment by TwofoldRisk** [comment]
Hi there, no in the selve windows tool it isn‘t possible to set a defined set. Not the se pro (which does in home assistant) nor the se com pro

**Comment by Android551** [comment]
Hi, 
the selve Windows tool is required to set up the RF-USB gateway. If I have set up the shutters and perhaps groups there, the gateway should be able to move the shutters via homeassistant. Can I then only control up, down and stop? In ioBroker, the shuttercontroll adapter gives the selve adapter the command to move to 60%. This works perfectly.

**Comment by TwofoldRisk** [comment]
Ok now I understand. The se com works very well, with set to percent commands to. But the se com pro seems to ignore or can‘t handle this commands because nothing happens. With se com pro only up down amd stop works.

**Comment by Android551** [comment]
I have selve SEL PLUS shutter motors. Does it work, too?

**Comment by TwofoldRisk** [comment]
That is a question for kannix because I‘m unable to say because I haven‘t one.

**Comment by Android551** [comment]
@Kannix2005 Can you answer the questions, please?

**Comment by Android551** [comment]
The Selve SE Pro is the successor to the Selve SEL-Plus

**Comment by frankfrommelt** [comment]
As far as I know it is depending on the protocol. Commeo is bi-directional
and supports status feedback e.g. position reporting.
The older Iveo protocol is uni-directional and does not provide feedback on
positions.
So, what protocol is used by your motors?

--- Original Nachricht ---
Von: Android551 ***@***.***
Datum: 6. März 2025 19:28:23
An: Kannix2005/homeassistant-selve ***@***.***
CC: Subscribed ***@***.***
Betreff: Re: [Kannix2005/homeassistant-selve] Set Position for Selve
SE-Pro-Com doesn't work (Issue #28)

**Comment by Android551** [comment]
My roller shutter motors (Selve SEL-PLUS) use the commeo protocol. This means that positions are reported back to the gateway.

**Comment by Kannix2005** [comment]
Sooo.. Everything works? 😁

**Comment by Kannix2005** [comment]
Thank you :)
